bootstrap/041_quotas.sql

```diff
@@ -1,16 +1,16 @@
 
-create table if not exists internal.quota_task_history(start_time timestamp_ltz, end_time timestamp_ltz, credits_used object, outcome string);
+create table if not exists internal.quota_task_history(start_time timestamp, end_time timestamp, credits_used object, outcome string);
```

Comment:
This is a breaking change. We cannot do a normal migration we cannot `alter table` to change the datatype from `timestamp_ltz` to `timestamp`.

Currently, the only accounts which might have this table are sundeck accounts, so I think it best to not try to maintain some migration process.

Reply:
ive not thought too much about it but how are we handling timezones here?

Reply:
This is the last piece I have to figure out, and I'm not entirely sure what is "correct" to do. So far, I've just been making sure that the timestamps are consistent (i.e. the hourly bucket matches when the task ran)